README.md

```diff
@@ -148,3 +148,9 @@ for (User tmpUser : users) {
 ## Contribute to the Project
 
 Take a look at the (contribution guide)[CONTRIBUTING.md] and the [build instructions wiki](https://github.com/okta/okta-sdk-java/wiki/Build-It) (though just cloning the repo and running `mvn install` should get you going).
+
+## Versioning
```

Comment:
@dogeared @mraible @laura-rodriguez thoughts on this topic?
I don't think this is likely to effect anyone, but I've added a couple non-`default` methods to an interface in this commit, so I feel like it needs to be stated someplace.

Reply:
nit: affect not effect

Reply:
it looks like @nbarbettini is OK with this, so I am :)